### PR TITLE
[docs] Added note how to install tox on Debian

### PR DESCRIPTION
**Pull request by chrysle**
### Thanks for contributing, make sure you address all the checklists (for details on how see [development documentation](https://virtualenv.pypa.io/en/latest/development.html#development))

Just stumbled over #1873 again and thought it would be helpful to add a note about that to the docs.

- [ ] ran the linter to address style issues (`tox -e fix`)
- [x] wrote descriptive pull request text
- [ ] ensured there are test(s) validating the fix
- [x] added news fragment in `docs/changelog` folder
- [x] updated/extended the documentation

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -31,6 +31,10 @@ virtualenv is a command line application written in Python. To work on it, you'l
 - :pypi:`tox`: to automatically get the projects development dependencies and run the test suite. We recommend
   installing it using `pipx <https://pipxproject.github.io/pipx/>`_.
 
+.. note::
+
```
Comment (gaborbernat) [inline docs/development.rst:35]: I don't think this is the case and we already voted against adding os special installation. pipx should work

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: docs/development.rst (hunk 1 above)
Comment (chrysle) [reply docs/development.rst:35]: My intention was to warn Debian users **not** to use the os specific installation but to use `pip` and `pipx` (or zipapp and other). I mean that with _alternative installation methods_.

I had installed the debian package `tox` (which installs `python3-virtualenv` as a dependency) and tried to invoke the development environment (`tox -e dev`) with the following traceback: 

```
.tox create: /home/user/repos/virtualenv/.tox/.tox
ERROR: invocation failed (exit code 1), logfile: /home/user/repos/virtualenv/.tox/.tox/log/.tox-0.log
================================================================================= log start ==================================================================================
ModuleNotFoundError: No module named 'virtualenv.activation.xonsh'

================================================================================== log end ===================================================================================
ERROR: InvocationError for command /usr/bin/python3 -m virtualenv --no-download --python /usr/bin/python3 .tox (exited with code 1)
``` 
Don't you think a warning would be helpful if other developers on Debian run into that issue?

Comment (gaborbernat) [reply docs/development.rst:35]: That like could be useful. But

Comment (chrysle) [reply docs/development.rst:35]: > That like could be useful. But

But? Would you suggest any changes?

Comment (gaborbernat) [reply docs/development.rst:35]: I meant not documentation but runtime error to be better 🥰

Comment (chrysle) [reply docs/development.rst:35]: Hm, but while this issue is still present (and I don't think I'll be able to fix it), wouldn't it be good to spread the information? 

Maybe someone will be inspired by the note to fix it!  😉

Comment (gaborbernat) [reply docs/development.rst:35]: They can get inspired by reading the bug tracker? 🙂

Comment (chrysle) [reply docs/development.rst:35]: Well, just trying to help (and doing some persuasive work 😉).

Many developers may run into that issue and not know what to do. If they find out what cause this has in the main development docs (which surely almost everyone will read), this'll bear no problem. I personally feel reading docs more comfortable than searching for old, stale (and potentially locked) issues.

Maybe we could add the note anyway **and** transer the problem to a new issue, which we could link to?

Comment (gaborbernat) [reply docs/development.rst:35]: I'd rather fix the underlying issue than patch the documentation like this 🤷‍♂️

Comment (chrysle) [reply docs/development.rst:35]: OK, I give up. Will open an issue.

**Comment by chrysle** [comment]
I have opened #2557

**Pull request closed without merging**